packages/bot-web-ui/src/components/quick-strategy.jsx

```diff
@@ -62,77 +64,141 @@ const QuickStrategyForm = ({
-                                                    <Icon icon={`IcUnderlying${selected_symbol.value}`} size={24} />
-                                                )
-                                            }
-                                        />
+                                        <>
+                                            {!is_mobile ? (
+                                                <Autocomplete
+                                                    {...field}
+                                                    autoComplete='off'
+                                                    className='quick-strategy__dropdown quick-strategy__leading'
+                                                    type='text'
+                                                    label={localize('Asset')}
+                                                    list_items={symbol_dropdown}
+                                                    onHideDropdownList={() => {
+                                                        onHideDropdownList('symbol', values[field.name], setFieldValue);
+                                                    }}
+                                                    onItemSelection={({ value }) => {
+                                                        onChangeDropdownItem('symbol', value, setFieldValue);
+                                                    }}
+                                                    onScrollStop={() => onScrollStopDropdownList('symbol')}
+                                                    leading_icon={
+                                                        selected_symbol.value && (
+                                                            <Icon
+                                                                icon={`IcUnderlying${selected_symbol.value}`}
+                                                                size={24}
+                                                            />
+                                                        )
+                                                    }
+                                                />
+                                            ) : (
+                                                <Dropdown
+                                                    placeholder={localize('Asset')}
+                                                    is_align_text_left
+                                                    list={symbol_dropdown}
+                                                    name='symbol'
+                                                    value={selected_symbol.value}
+                                                    onChange={e => {
+                                                        onChangeDropdownItem('symbol', e.target.value, setFieldValue);
+                                                    }}
+                                                />
```

Comment:
in order to design  consistency you should pas icon and Dropdown structure to your Dropdown list

Reply:
`Dropdown` doesn't accept icon, and we should ask design to apply changes for `native`.